Solutions/Network Threat Protection Essentials/Hunting Queries/UseragentExploitPentest.yaml

```diff
@@ -1,6 +1,8 @@
 id: df75ac6c-7b0b-40d2-82e4-191c012f1a07
 name: Exploit and Pentest Framework User Agent
 description: |
+    'This query detects suspicious user agent strings used by exploit and pen test frameworks. It's based on sigma rules and focuses on data sources containing the UserAgent field.'
```

Comment:
This query detects suspicious user agent strings used by exploit and pen test frameworks. (This is good enough)

Reply:
Resolved